lib/ansible/module_utils/facts/system/chroot.py

```diff
@@ -22,7 +22,18 @@ def is_chroot():
             is_chroot = my_root.st_ino != proc_root.st_ino or my_root.st_dev != proc_root.st_dev
         except Exception:
             # I'm not root or no proc, fallback to checking it is inode #2
-            is_chroot = (my_root.st_ino != 2)
+            fs_root_ino = 2
+
+            if module is not None:
+                # on Btrfs, the first-created subvolume (root) has inode #256
+                stat_path = module.get_bin_path('stat')
```

Comment:
don't fail on missing 'stat', return None/N\A on the fact

Reply:
If `stat` is missing, it will fall back to the current behavior. `required=False` by default in `get_bin_path`.

Reply:
ah, for some reason i thought 'True' was default, ignore me. ....